crates/uv-python/src/download-metadata-minified.json

```diff
@@ -0,0 +1 @@
,"minor":10,"patch":2,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20220227/cpython-3.10.2%2B20220227-x86_64_v4-unknown-linux-gnu-install_only.tar.gz","sha256":"0493e89aae5b95b17f3092402cd7b6516494cad817aff68cc2988a4c065ae1ab","variant":null},"cpython-3.10.2-windows-i686-none":{"name":"cpython","arch":{"family":"i686","variant":null},"os":"windows","libc":"none","major":3,"minor":10,"patch":2,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20220227/cpython-3.10.2%2B20220227-i686-pc-windows-msvc-shared-install_only.tar.gz","sha256":"5321f8c2c71239b1e2002d284be8ec825d4a6f95cd921e58db71f259834b7aa1","variant":null},"cpython-3.10.2-windows-x86_64-none":{"name":"cpython","arch":{"family":"x86_64","variant":null},"os":"windows","libc":"none","major":3,"minor":10,"patch":2,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20220227/cpython-3.10.2%2B20220227-x86_64-pc-windows-msvc-shared-install_only.tar.gz","sha256":"a1d9a594cd3103baa24937ad9150c1a389544b4350e859200b3e5c036ac352bd","variant":null},"cpython-3.10.0-darwin-aarch64-none":{"name":"cpython","arch":{"family":"aarch64","variant":null},"os":"darwin","libc":"none","major":3,"minor":10,"patch":0,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20211017/cpython-3.10.0-aarch64-apple-darwin-pgo%2Blto-20211017T1616.tar.zst","sha256":null,"variant":null},"cpython-3.10.0-darwin-x86_64-none":{"name":"cpython","arch":{"family":"x86_64","variant":null},"os":"darwin","libc":"none","major":3,"minor":10,"patch":0,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20211017/cpython-3.10.0-x86_64-apple-darwin-pgo%2Blto-20211017T1616.tar.zst","sha256":null,"variant":null},"cpython-3.10.0-linux-aarch64-gnu":{"name":"cpython","arch":{"family":"aarch64","variant":null},"os":"linux","libc":"gnu","major":3,"minor":10,"patch":0,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20211017/cpython-3.10.0-aarch64-unknown-linux-gnu-lto-20211017T1616.tar.zst","sha256":null,"variant":null},"cpython-3.10.0-linux-i686-gnu":{"name":"cpython","arch":{"family":"i686","variant":null},"os":"linux","libc":"gnu","major":3,"minor":10,"patch":0,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20211017/cpython-3.10.0-i686-unknown-linux-gnu-pgo%2Blto-20211017T1616.tar.zst","sha256":null,"variant":null},"cpython-3.10.0-linux-x86_64-gnu":{"name":"cpython","arch":{"family":"x86_64","variant":null},"os":"linux","libc":"gnu","major":3,"minor":10,"patch":0,"prerelease":"","url":"https://github.com/astral-sh/python-build-standalone/releases/download/20211017/cpython-3.10.0-x86_64-unknown-linux-gnu-pgo%2Blto-20211017T1616.tar.zst","sha256":null,"variant":null},"cpython-3.10.0-windows-i686-none":{"name":"cpython","arch":{"family":"i686","v
```

Comment:
We'll have to make sure we regenerate this before merging, in case any releases have happened in the interim.

Reply:
(changes have already occurred yeah)